src/components/LibraryDetailContainer.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
```

Comment:
Do you need this component because you need to pass the store to call it for the child component? I think that was only the top level case but any child should work, no?

Reply:
That's why I have it, but I'll try getting rid of it and see what happens...

Reply:
You called it; component is not needed :)